sample.env

```diff
@@ -8,3 +8,6 @@
 
 # Comma separated list of admins
 #SPOTSEEKER_AUTH_ADMINS=javerage,demo_user
+
+# URL to sync techloan from sync_techloan management command
+#SPOTSEEKER_TECHLOAN_URL=https://asa-django-01.asa.uw.edu/techloan/api/v2/type/?embed=availability&embed=class
```

Comment:
Let's set the value to something exampleish, you might go ahead and keep the /techloan/api... etc.